pallets/gear/src/schedule.rs

```diff
@@ -56,7 +56,7 @@ pub const INSTR_BENCHMARK_BATCH_SIZE: u32 = 500;
 // To avoid potential stack overflow problems we have a panic in sandbox in case,
 // execution is ended with stack overflow error. So, process queue execution will be
 // stopped and we will be able to investigate the problem and decrease this constant if needed.
-pub const STACK_HEIGHT_LIMIT: u32 = 18_369;
+pub const STACK_HEIGHT_LIMIT: u32 = 36_743;
```

Comment:
Why?

Reply:
@breathx you should interpret this stack size as virtual, not real. 

Previously `wasm-instrument` did the following:
```wasm
(func $f
  call $f ;; global_stack_height += 2
)
```
i.e. the actual stack size is `18_369 / 2 = 9184` (wasmer stack size on windows)

Now `wasm-instrument` does `global_stack_height += 4` every time you call to `$f` plus `4` for the thunk call overhead.

Previous formula: `virtual_stack_size = 2 * count_of_f_calls`
Current formula: `virtual_stack_size = 4 * count_of_f_calls + 4`

```
18_369 // 2 = 9184
(36_743 - 4) // 4 = 9184
```

Grisha has reviewed PR and this has already been discussed